BUILDING.md

```diff
@@ -163,7 +163,7 @@ Binaries at <https://nodejs.org/download/release/> are produced on:
 | Binary package          | Platform and Toolchain                                                                                      |
 | ----------------------- | ----------------------------------------------------------------------------------------------------------- |
 | aix-ppc64               | AIX 7.2 TL04 on PPC64BE with GCC 10                                                                         |
-| darwin-x64              | macOS 10.15, Xcode Command Line Tools 11 with -mmacosx-version-min=10.15                                    |
+| darwin-x64              | macOS 11, Xcode Command Line Tools 12 with -mmacosx-version-min=10.15                                       |
 | darwin-arm64 (and .pkg) | macOS 11 (arm64), Xcode Command Line Tools 12 with -mmacosx-version-min=10.15                               |
```

Comment:
This should probably be amended to just `Xcode`? We're using full Xcode to do the notarization (probably `Xcode Command Line Tools` used to be correct and we didn't update).